packages/config-resolver/src/NonAwsEndpointsConfig.ts

```diff
@@ -0,0 +1,45 @@
+// TODO: Create a .spec.ts for this
+import { Endpoint, Provider, UrlParser } from "@aws-sdk/types";
+
+export interface NonAwsEndpointsInputConfig {
```

Comment:
Can we call this `CustomEndpoint` or `StaticEndpoint` rather than `NonAwsEndpoint`?

Reply:
Yeah I asked this in my description:
> EndpointsConfig and AwsEndpointsConfig may be better than NonAwsEndpointsConfig and EndpointsConfig. Is it fine to rename these? Or other suggestions on names.

Can we do EndpointsConfig and AwsEndpointsConfig? If not, a name different than NonAwsEndpointsConfig definitely needed. I like CustomEndpoint; it has a `isCustomEndpoint:true` too.

Reply:
It would probably  be a breaking change to rename the existing one is the problem